### DISCUSSION TITLE
How to filter out old files

### DISCUSSION THREAD

**Discussion by QuAzI** [BurntSushi/ripgrep#2922] [Q&A] [answered]
Is there any way to rg all files recursively, but only if they are not older than 3 days or 1 hour or changed between some period?

**Comment by BurntSushi** [comment, marked as answer]
ripgrep doesn't have facilities for filtering by timestamp. It might grow them some day, but for now, you'll need to use some other tool. e.g., `find ./ some-filter-options -print0 | xargs -0 rg pattern-to-search-for`.
